tests/installation/ipxe_install.pm

```diff
@@ -103,8 +103,9 @@ sub set_bootscript {
             $cmdline_extra .= " ssh=1 sshpassword=$testapi::password ";    # trigger ssh-text installation
         }
         else {
-            $cmdline_extra .= " vnc=1 VNCPassword=$testapi::password ";    # trigger default VNC installation
-        }
+	    # trigger default graphical installation via SSH or VNC
+            $cmdline_extra .= " sshd=1 sshpassword=$testapi::password vnc=1 VNCPassword=$testapi::password ";
```

Comment:
Isn't this normally just `ssh=1` (as documented on https://doc.opensuse.org/documentation/leap/startup/html/book-startup/cha-boot-parameters.html#sec-boot-parameters-list-remote-access)? I'm not saying `sshd=1` is wrong because I don't really know this test scenario. However, it doesn't seem to be very in-line with the comment you've added judging by the linked documentation.

Reply:
I've just seen @Julie-CAO 's review. Not sure what the conclusion from her comments is, though. In doubt, you can likely ignore my review comment.

Reply:
Looking back when it still worked, we had sshd=1, so  @Julie-CAO is right on this.

Reply:
I tried `ssh=1` and `sshd=1` out when I worked on my previous PR. When `ssh=1` is used, you have to open a ssh console and type `yast.ssh` to begin with SLE installation. For our TEXTMODE tests with needle matching(without autoyast), we just do it this way because ssh console is much more stable than sol console. While for our tests with autoyast installation, we  don't care about console so much, so we remove `ssh=1` to use sol console, `sshd=1` being used at the same time is for enable debugging or post_fail_hook with sshd.service started during installation.  I hope I clealy speak out the whole story.

Reply:
So then ssh=1 is what we actually want. We used sshd=1 and it also worked, but was not correct.